helm/nginx-ingress-controller-app/tests/abs/test_basic_cluster.py

```diff
@@ -61,6 +63,28 @@ def wait_for_ic_deployment(kube_cluster: Cluster) -> List[pykube.Deployment]:
     return deployments
 
 
+def try_ingress(port, host, expected_status):
+    retries = 5
+    last_status = 0
+    while retries != 0:
+        logger.info(f"trying GET http://127.0.0.1:{port}/ with Host: {host}")
+        try:
+            r = requests.get(f"http://127.0.0.1:{port}/", headers={"Host": host})
```

Comment:
you cannot use the Ingress hostname? https://github.com/giantswarm/nginx-ingress-controller-app/pull/218/files#diff-d7ad5a5324de0bb9fbde4275d17b74a41126017ca2576b1c4a3a65a6cc820fcdR93

Reply:
I think not, this sends a GET request to the NodePort of the kind cluster, so unless we deploy also some kind of DNS the ingress hostname is not known. The `Host` header tells the nginx inside the ingress controller to which host this request should be routed.

Reply:
ah ok I got it now